CNV-18298: force stop release note (#65032)
openshift/openshift-docs: virt/release_notes/virt-4-14-release-notes.adoc

// CNV-29848: Release: Mechanism to manage DP/TP features in the UI
* Cluster administrators can enable or disable link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features on the link:https://docs.openshift.com/container-platform/4.14/virt/getting_started/virt-web-console-overview.html#overview-settings-cluster_virt-web-console-overview[*Settings*] tab on the *Virtualization* -> *Overview* page.

//CNV-18298 forced stop

> * You can now *Force stop* an unresponsive VM from the xref:../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-page_virt-web-console-overview[action menu]. If you select *Stop*, *Force stop* replaces *Stop* in the action menu.

Reviewer: * I think this would read better to write just "force stop" normally in the first sentence.
* Also, maybe this is clearer if I could actually see what's happening in the UI, but I don't understand what it means about "Force stop" replacing "Stop" in the action menu. So, if you want to force stop something, you have to hit "Stop" first and then "Force Stop"? If that's the case, would probably help to be more explicit. Adding a suggestion below, but feel free to adjust/ignore.

```suggestion
* You can now force stop an unresponsive VM from the xref:../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-page_virt-web-console-overview[action menu]. To force stop a VM, select *Stop* and then *Force stop* from the action menu.
```

Author: @bergerhoffer That's correct - the Force Stop option only shows up if you've already chosen Stop from the actions menu, and it shows up in the menu where the Stop option previously was.
